File: soobinJung/Programmers/level1/1.java
class Solution {
public int solution(int[] numbers) {
int answer = 45;

        for(int number : numbers) {
            answer -= number;
        }

        return answer;
    }

}

<!--
겪었던 문제점
처음에
Arrays.stream(numbers)
IntStream.of(a).anyMatch(x -> x == 4)
이거 썼는데 import 해줘야 하는지 모르고 계속 실행하다가 안되서 헤맸다.
-->

File: yoojinYang/Programmers/2/1.py
# 프로그래머스 코딩테스트 연습 > 연습문제
# level 1
# 가운데 글자 가져오기

# 내 풀이
def solution(s):
    if(len(s)%2!=0):
        answer = s[len(s)//2]
    else:
        answer = s[len(s)//2-1]+s[len(s)//2]
    return answer


# 신박한 풀이
def solution(s):
    return s[(len(s)-1)//2:len(s)//2+1]